sql/moz-fx-data-shared-prod/telemetry_derived/firefox_use_counters_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Firefox Use Counters
+description: |-
+  Contains use counter rates by day, country, version, and platform; only shows data for those with at least 5000 clients reporting that day.
+owners:
+- kwindau@mozilla.com
+labels:
+  incremental: true
+  owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: use_counter_analysis
+bigquery:
+  time_partitioning:
+    type: day
+    field: 'received_date'
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields: ['metric']
+references: {}
```

Comment:
```suggestion
```

Reply:
just removed that, thanks for the suggestion!